.circleci/config.yml

```diff
@@ -184,6 +184,8 @@ jobs:
               -d opcache.enable_cli=1 \
               -d opcache.jit_buffer_size=16M \
               -d opcache.jit=tracing \
+              -d zend_test.observer.enabled=1 \
+              -d zend_test.observer.show_output=0 \
```

Comment:
Might it make sense to test this on x86_64 in nightly instead? E.g. `.github/nightly_matrix.php`, under `get_matrix_include()`, `_VARIATION`. That one also runs without opcache, with opcache and with JIT(s).

Reply:
I personally would like to see it run in a push-job, with --repeat 2, and not nightly only.

Reply:
Alright, lgtm then :slightly_smiling_face:

Reply:
@bwoebi Do you really like to run this on ARM only?

Reply:
@dstogov I suppose we can add it on one x86 job as well. Makes sense.

Reply:
@bwoebi You can add it to either the nightly repeat or variation job.

Reply:
I also think it make more sense to setup specialized nightly jobs.

Reply:
@dstogov The variation job enables multiple debug checks, I think it's not too out of place there. I'm not sure there's a big benefit in testing each separately. Usually it's pretty clear what is causing the issue, from the error message.

Reply:
I don't know what is "variation job". Do you mean "Test", "Test Tracing JIT", etc? Of course "Test Observer" and "Tesr Observer + Function JIT" may be add as this kind of sub-jobs.

Reply:
https://github.com/php/php-src/blob/dad2d561e83fcdb7b7b204dc981b52f557228dab/.github/nightly_matrix.php#L61-L70

This job runs tests with additional assertions.